src/aat/resources/features/F-064/S-218.td.json

```diff
@@ -0,0 +1,24 @@
+{
+  "_guid_": "S-218",
+  "_extends_": "F-064_Test_Data_Base",
+  "title": "must return negative response when request provides authentic credentials without authorised access",
```

Comment:
The scenario title mismatch.

Reply:
The input variable is wrong for creating the expected scenario. This is for scenario 219.